src/3box.js

```diff
@@ -173,10 +173,18 @@ class ThreeBox {
     }
   }
 
+  /**
+   * Uninitializes the 3box. Should be called after you are done with
```

Comment:
It can be unclear what `Uninitializes the 3box` actually means.

Reply:
@c-castillo do you have a suggestion of what we could say instead? Basically we are just closing the orbitdb instance correctly.

Reply:
`Closes the 3box instance` or more technical `Closes the underlying storage instance of the 3box`
Maybe the first one is better

Reply:
Ok, thanks!